File: app/src/test/java/pl/piotrskiba/angularowo/TextUtilsTests.java
package pl.piotrskiba.angularowo;

import org.junit.Assert;
import org.junit.Test;

import pl.piotrskiba.angularowo.utils.TextUtils;

public class TextUtilsTests {

    @Test
    public void normalize_specialChars_normalized() {
        Assert.assertEquals("eoaszzcnaY", TextUtils.normalize("ęóąśłżźćń¥ƒáÝ‰"));
    }

    @Test
    public void normalize_normalizedText_notChanged() {
        String sentence = "Already normalized sentence.";
        Assert.assertEquals(sentence, TextUtils.normalize(sentence));
    }
}

File: app/src/test/java/pl/piotrskiba/angularowo/TextUtilsTests.kt
package pl.piotrskiba.angularowo

import org.junit.Assert
import org.junit.Test
import pl.piotrskiba.angularowo.utils.TextUtils.normalize

class TextUtilsTests {
    @Test
    fun normalize_specialChars_normalized() {
        Assert.assertEquals("eoaszzcnaY", normalize("ęóąśłżźćń¥ƒáÝ‰"))
    }

    @Test
    fun normalize_normalizedText_notChanged() {
        val sentence = "Already normalized sentence."
        Assert.assertEquals(sentence, normalize(sentence))
    }
}